src/screens/Customers/CustomerMain.tsx

```diff
@@ -73,29 +73,25 @@ const labels: FilterByLabel = {
 }
 
 function CustomerMain(props: CustomerMainProps) {
-  const { classes, customers } = props;
+  const { classes } = props;
   const [filteredCustomers, setFilteredCustomers] = useState<CustomerRecord[]>([]);
   const [filteredCustomersAlt, setFilteredCustomersAlt] = useState<CustomerRecord[]>([]);
-  const [fullCustomers, setFullCustomers] = useState<CustomerRecord[]>([]);
   const [allCustomersTrie, setAllCustomersTrie] = useState<TrieTree<CustomerRecord>>(new TrieTree('name'));
   const [sortBy, setSortBy] = useState<SortBy>(SortBy.NAME);
   const [filterBy, setFilterBy] = useState<FilterBy>(FilterBy.ACTIVE_STATUS);
   const [sortAndFilter, setSortAndFilter] = useState<string[]>([])
   const [filterLabels, setFilterLabels] = useState<string[]>(labels["ACTIVE_STATUS"]);
   const [searchValue, setSearchValue] = useState<string>("");
+  const fullCustomers = useSelector(selectAllCustomersArray) || [];
```

Comment:
This breaks when it's `[]` -- you might have to use `[] as CustomerRecord[]` instead

Reply:
Alternatively I think you can add `fullCustomers: CustomerRecord[]` based on what you did in `Home`

Reply:
Did the latter, thanks!